src/ck-ldb/LBManager.C

```diff
@@ -484,7 +485,7 @@ void LBManager::initnodeFn()
   _registerCommandLineOpt("+LBSameCpus");
   _registerCommandLineOpt("+LBUseCpuTime");
   _registerCommandLineOpt("+LBOff");
-  _registerCommandLineOpt("+LBCommOff");
```

Comment:
We may want to leave the +LBCommOff option as well (as a no-op) so that if anyone has a script passing it it will still get parsed out by the runtime and not passed through to the application with a warning about 'unknown CLA beginning with a +'